<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>facePvms</class>
 <widget class="QWidget" name="facePvms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>facePvms</string>
  </property>
  <widget class="QLabel" name="company_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">人脸识别分析软件face-pvms
font: 9pt &quot;楷体&quot;;
font: 24pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;人脸识别分析软件 face-pvms(试用版0.04)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabelWidget_operation">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>70</y>
     <width>201</width>
     <height>511</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <widget class="QWidget" name="operation">
    <attribute name="title">
     <string>操作</string>
    </attribute>
    <widget class="QLabel" name="label_devname">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string> 设备序号</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_deviceNumber">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushbutton_device_config">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>配置</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_configArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>181</width>
       <height>361</height>
      </rect>
     </property>
     <property name="title">
      <string>配置区域</string>
     </property>
     <widget class="QLabel" name="label_ipaddr">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>设置IP：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_interval">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>提取间隔</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_sec">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> 秒</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_usr">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_password">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>110</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>密码：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_port">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>端口号：</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_intervalSec">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_save">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>290</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string> 确认</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_devicename">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>设备名：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_devicename">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>230</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_account">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ipaddr">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_port">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_deviceOpera">
      <item>
       <widget class="QPushButton" name="pushbutton_device_open">
        <property name="text">
         <string>打开设备</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_device_close">
        <property name="text">
         <string>关闭设备</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="inquery">
    <attribute name="title">
     <string>查询</string>
    </attribute>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_deviceName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>设备名</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_starttime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>开始时间</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_endTime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>131</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>结束时间</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_search">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_startCanlendar" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="widget_endCanlendar" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>存储</string>
    </attribute>
    <widget class="QLabel" name="label_imageName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>存储路径：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_imagePath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_pathSelect">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>选择</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_storageDateName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>存储时间:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_storagedate">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_day">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>天</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_storageConfirm">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>651</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_leftLayout">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget_result">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_display">
       <attribute name="title">
        <string>摄像头显示</string>
       </attribute>
       <widget class="QLabel" name="label_DisplayCam">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>640</width>
          <height>480</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;显示区域&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_searchResult">
       <attribute name="title">
        <string>查询结果</string>
       </attribute>
       <widget class="QListWidget" name="listWidget_imagelist">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>640</width>
          <height>480</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_curdevice">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>590</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_curname">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>当前工作设备数量：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_systimename">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>590</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> 当前时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_systime_show">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>590</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>spinBox_deviceNumber</tabstop>
  <tabstop>pushbutton_device_config</tabstop>
  <tabstop>pushbutton_device_open</tabstop>
  <tabstop>pushButton_device_close</tabstop>
  <tabstop>spinBox_intervalSec</tabstop>
  <tabstop>lineEdit_account</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>lineEdit_ipaddr</tabstop>
  <tabstop>lineEdit_port</tabstop>
  <tabstop>lineEdit_devicename</tabstop>
  <tabstop>pushButton_save</tabstop>
  <tabstop>tabelWidget_operation</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>pushButton_search</tabstop>
  <tabstop>tabWidget_result</tabstop>
  <tabstop>lineEdit_imagePath</tabstop>
  <tabstop>pushButton_pathSelect</tabstop>
  <tabstop>spinBox_storagedate</tabstop>
  <tabstop>pushButton_storageConfirm</tabstop>
  <tabstop>listWidget_imagelist</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
